arm-sql/2015-05-01-preview/examples/DatabaseAutomaticTuningGet.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "subscriptionId": "c3aa9078-0000-0000-0000-e36f151182d7",
+    "resourceGroupName": "default-sql-onebox",
+    "serverName": "testsvr11",
+    "databaseName": "db1",
+    "api-version": "2015-05-01-preview"
```

Comment:
out of curiosity, is this service considered as preview on your end, so you're following the convention of using "-preview" on the api version?